crafter-search-provider/solr/configsets/crafter_configs/conf/managed-schema

```diff
@@ -175,6 +175,7 @@
     <dynamicField name="*_d"  type="pdouble" indexed="true"  stored="true"/>
     <dynamicField name="*_ds" type="pdoubles" indexed="true"  stored="true"/>
     <dynamicField name="*_dmv" type="pdoubles" indexed="true"  stored="true"/>
+    <dynamicField name="*_raw" type="string" indexed="true"  stored="true"/>
```

Comment:
this shouldn't be a string, this should be text. The body still needs to be analyzed/tokenized.